src/modules/ROOT/pages/changelog/202303/00.json

```diff
@@ -0,0 +1,800 @@
+      "what": [
+        "new style configuration for the file page block and file record field.",
+        "Users can now change the height, width, borders, margins and the background color of their file attachment."
+      ],
+      "why": [
+        "to provide users with more customization and styling option on viewing images."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/533/commits/ccd90d74d4945e96d21281c68bfe9f18bc394844"
+      ]
+    },
+    {
+      "what": [
+        "support for opening a record in a new tab or in a modal."
+      ],
+      "why": [
+        "to make record entry a bit more intuitive, since the user will remain on the record list page while doing the changes."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/524/commits/2647e3b8d72afa31e79457f3cd985ae0b7fa31fc",
+        "https://github.com/cortezaproject/corteza/pull/524/commits/950d305a5a8fb79ee84599989a399af7d028f599"
+      ]
+    },
+    {
+      "what": [
+        "the support for cloning and coping page blocks in page builders."
+      ],
+      "why": [
+        "to allow users to duplicate and re-use page blocks more easily."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/582/commits/7f377109d0b46ade1b04857569622829f917321b"
+      ]
+    },
+    {
+      "what": [
+        "an option to magnify certain page blocks when viewing a public {APP_NAME_COMPOSE} page.",
+        "A select dropdown has been added in the general tab of page block configuration modal for calendar, chart, content, comments, file, iframe, record list, report, and record revisions page blocks.",
+        "The user can select between Fullwidth or Modal select option, and when enabled, the user will see a magnify icon at the top of the block."
+      ],
+      "why": [
+        "to help users finding the info they are looking for much faster and easier with the option to magnify certain page blocks."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/576/commits/1a836ddf2c0c9e5892b791ba201032d5d8a37365"
+      ]
+    },
+    {
+      "what": [
+        "a spinner to indicate when an action in template editor is submitted."
+      ],
+      "why": [
+        "to have an UI consistency, because it was difficult to tell when your changes were being submitted."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/708/commits/7a4a0dfc87756194f66365f1aad26b1447b4b2b5"
+      ]
+    },
+    {
+      "what": [
+        "a module field description when configuring a record list page block to inform the users that if no fields are selected, the default values will be shown."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/751/commits/b4d5c7fd7cdef17c847717e2944fc669c213204d"
+      ]
+    },
+    {
+      "what": [
+        "ability to serve data as response in an integration gateway route."
```

Comment:
Can we mention how this is done? Do we have a new processing bit? Is that for use with workflows?